cadctl/cmd/investigate/investigate.go

```diff
@@ -131,20 +129,6 @@ func run(_ *cobra.Command, _ []string) error {
 	return alertInvestigation.Run(investigationResources)
 }
 
-// getInvestigation will return the investigation function for the identified alert
-func getInvestigation(alertTitle string) *investigation.Investigation {
```

Comment:
Moved this to pkg/investigations/mapping: this is re-used by the interceptor so we don't have to add the investigation mapping in two places.

Reply:
Is a new package needed? The function could live in the investigations package and the investigations package could be the one imported.

Reply:
I think it makes sense to split them as we're splitting the usage from the implementation. I'd expect barely any modifications to the `investigation` package in the future, whereas the `mapping` package will constantly be modified. We might want different code reviewers for those in the future.

Reply:
alright